src/git.ts

```diff
@@ -0,0 +1,122 @@
+import * as github from "@actions/github";
+import { Octokit } from "@octokit/rest";
+import * as core from "@actions/core";
+import parseDiff, { File as DiffFile } from "parse-diff";
+import { ReviewComment } from "./types";
+
+const GITHUB_TOKEN = core.getInput("GITHUB_TOKEN");
+const octokit = new Octokit({ auth: GITHUB_TOKEN });
```

Comment:
Consider validating the GITHUB_TOKEN before using it to ensure it is not undefined or empty, which could lead to runtime errors.